source/_posts/2019-10-08-introducing-f-twelve-an-open-source-dev-console.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Introducing F-Twelve, an Open Source Dev Console"
+author: Patrick Gross
+date: 2019-10-08
+tags: [engineering, oss]
+---
+
+# Introducing F-Twelve, an Open Source Dev Console
```

Comment:
not necessary since the title will already be displayed on the post

Reply:
![image](https://user-images.githubusercontent.com/8029578/66502653-53469880-ea8b-11e9-9869-648f255acb48.png)
Updated